.github/workflows/install-latest-to-ubuntu-20-04.yml

```diff
@@ -8,8 +8,47 @@ jobs:
         uses: actions/checkout@v3
       - run: echo "💡 The ${{ github.repository }} repository has been cloned to the runner."
       #Get latest version
+      - run: wget https://raw.githubusercontent.com/ant-media/Scripts/4ce6a9b5fe7e1f226178d84d26e2883289e3c482/install_ant-media-server.sh && chmod 777 install_ant-media-server.sh
```

Comment:
why don't we get from master branch or the branch having the same name?

There is a solution for this problem in travis side.
https://github.com/ant-media/Ant-Media-Server/blob/master/.travis.yml#L106